woocommerce/woocommerce-android
# Issue/3196 remove fulfilment code

Fixes #3196 

This PR removes unused order fulfilment and product screen related code. As part of this removal the below was done:
- DI clean-up
- Navigation graph clean-up
- Layout clean-up
- String clean-up
- Fragment clean-up
- View clean-up
- Presenter clean-up (including tests)
- Analytics clean-up

### To test

Open the `Orders` tab and navigate around. Make sure that orders are displayed as expected and nothing order-wise is broken.
